style.css

```diff
@@ -15,31 +15,39 @@ body {
 }
```

Comment:
- [ ] Minimum mobile screen size should be 375px. There are still errors in screen sizes between 375px and 468px. I would recommend fixing that to improve your UI design.

- Your design (mobile size: 375px):
<img width="205" alt="image" src="https://user-images.githubusercontent.com/77190203/171748769-0e2574d4-1dad-450d-aa00-6123299f3f81.png">

Reply:
Hi @medaminedev66! Thank you so much for the feedback. As you can see in the preview comments, I can't replicate the error on my side. Here I add some screenshots on different mobile resolutions. Could this be a Chrome issue?
I'll wait for your feedback to proceed because I only have one 'request review' left. If you could provide me with some tips, I'll be thankful. 


![Screenshot from 2022-06-02 18-36-52](https://user-images.githubusercontent.com/37988027/171749613-12434fda-9efa-43bd-8ca3-3a267f2cc136.png)

![Screenshot from 2022-06-02 18-36-38](https://user-images.githubusercontent.com/37988027/171749600-de8f821a-f88d-467f-8d05-3bad3f7997c2.png)

![Screenshot from 2022-06-02 17-39-11](https://user-images.githubusercontent.com/37988027/171749581-61ab0fa8-a94e-420b-8d2c-90e933520c69.png)

Reply:
Hi @rominana, the hamburger icon is not shown in the screenshots you provided. Simply ensure that the hamburger icon is there in the navbar and that your page is not scrolling horizontally.